components/news/Admin.vue

```diff
@@ -164,7 +168,7 @@ export default {
       this.$emit('update:selectedToEdit', null)
     },
     deleteNewsItem() {
-      this.deleteNews(this.selectedToDelete['.key'])
+      this.deleteNews(this.selectedToDelete)
```

Comment:
```suggestion
      this.deleteNews(this.selectedToDelete.id)
```

then pass the `id` in store/news.js